spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianRequestProperties.java

```diff
@@ -25,8 +30,14 @@ protected String getUrlLocalPart() {
         if (urlLocalPart == null) {
             urlLocalPart = StringUtils.removeStart(request.getPathInfo(), appPrefix);
         }
-
         return StringUtils.defaultString(urlLocalPart);
     }
 
+
+    @Override
+    public URI getUri() {
+        // ... enforce to use the configured base url of the underlying system e.g. Confluence instead of url of the http request.
```

Comment:
You are right about the UrlMode ... It was a mistake and I updated the related code part